### PR TITLE
[RHBA-394] refactor bpmsuite-7 openshift images to rhba-7

### PR DESCRIPTION
**Pull request by errantepiphany**
https://issues.jboss.org/browse/RHBA-394
Signed-off-by: David Ward <dward@redhat.com>

### CODE REVIEW

File: businesscentral/image.yaml [hunk 1]
```diff
@@ -0,0 +1,339 @@
...
+      example: "0.5"
+      description: "Deprecated.  See JAVA_MAX_MEM_RATIO."
+    - name: "INITIAL_HEAP_PERCENT"
+      example: "0.5"
+      description: "Deprecated.  See JAVA_INITIAL_MEM_RATIO."
+    - name: "PROBE_DISABLE_BOOT_ERRORS_CHECK"
+      example: "true"
+      description: "Disable the boot errors check in the probes."
+ports:
+    - value: 8443
```
Comment (spolti) [inline businesscentral/image.yaml]: shouldn't the port 8080 be listed here too?

File: controller/image.yaml [hunk 2]
```diff
@@ -0,0 +1,279 @@
...
+      example: "0.5"
+      description: "Deprecated.  See JAVA_MAX_MEM_RATIO."
+    - name: "INITIAL_HEAP_PERCENT"
+      example: "0.5"
+      description: "Deprecated.  See JAVA_INITIAL_MEM_RATIO."
+    - name: "PROBE_DISABLE_BOOT_ERRORS_CHECK"
+      example: "true"
+      description: "Disable the boot errors check in the probes."
+ports:
+    - value: 8443
```
Comment (spolti) [inline controller/image.yaml]: port 8080?

File: kieserver/image.yaml [hunk 3]
```diff
@@ -0,0 +1,409 @@
...
+      example: "0.5"
+      description: "Deprecated.  See JAVA_MAX_MEM_RATIO."
+    - name: "INITIAL_HEAP_PERCENT"
+      example: "0.5"
+      description: "Deprecated.  See JAVA_INITIAL_MEM_RATIO."
+    - name: "PROBE_DISABLE_BOOT_ERRORS_CHECK"
+      example: "true"
+      description: "Disable the boot errors check in the probes."
+ports:
+    - value: 8443
```
Comment (spolti) [inline kieserver/image.yaml]: port 8080?

File: templates/rhba70-kieserver-basic-s2i.yaml [hunk 4]
```diff
@@ -0,0 +1,307 @@
+kind: Template
+apiVersion: v1
+metadata:
+  annotations:
+    description: Application template for Red Hat Business Automation Execution Server 7.0 application built using S2I.
+    iconClass: icon-jboss
+    tags: rhba,jboss,xpaas
+    version: 1.4.0
```
Comment (spolti) [inline templates/rhba70-kieserver-basic-s2i.yaml]: wouldn't be better start the xpaas with the current version?

Comment (errantepiphany) [reply templates/rhba70-kieserver-basic-s2i.yaml]: Gonna keep it at 1.4.0 for now, and update it just before we tag.

Comment (spolti) [reply templates/rhba70-kieserver-basic-s2i.yaml]: +1

File: quickstarts/library-process/README.md [hunk 5]
```diff
@@ -0,0 +1,192 @@
+## Red Hat Business Automation Kie Server Quickstart
+
+This quickstart is intend to be used with the [RHBA Kie Server](https://github.com/jboss-container-images/rhba-7-openshift-image/tree/rhba70-dev/kieserver) image.
+
+## How to use it?
+
+To deploy the Library Process demo you can use the [rhba70-kieserver-s2i](https://github.com/jboss-container-images/rhba-7-openshift-image/blob/rhba70-dev/templates/rhba70-kieserver-s2i.yaml)
```
Comment (spolti) [inline quickstarts/library-process/README.md]: shouldn't be rhba70-kieserver-basic-s2i.yaml?
I guess that the next lines of this readme also needs to be update to reflect the template name.

Comment (errantepiphany) [reply quickstarts/library-process/README.md]: The readme has a lot of work that needs to be done to it. I am issuing this PR to get something that can be accessible for others to start building with, but will go back and work on docs in another PR. I'll fix this still, though.

**Pull request merged**